src/storage.rs

```diff
@@ -461,19 +568,20 @@ impl Session {
                 // hashchain already exists
                 let mut current_chain = value.clone();
 
-                let incoming_entry = match self.verify_signature(&signature) {
-                    Ok(public_key) => public_key,
+                // in the deforest approach we always have to check with "given key" if the signature is valid, not as with key transparency a previously added valid public key
```

Comment:
remove deforestation specific stuff - just mention this is because values are "arbitrary strings" and not strictly public keys